pkg/monitor/intervalcreation/node_test.go

```diff
@@ -10,6 +11,191 @@ import (
+				},
+				From: time.UnixMicro(1664269199857303).In(time.UTC),
+				To:   time.UnixMicro(1664269199857303).In(time.UTC),
+			},
+		},
+		{
+			name:    "reflector",
+			logLine: `Sep 27 08:59:59.853216 ci-op-747jjqn3-b3af3-f45pk-worker-centralus2-bdp5s kubenswrapper[2397]: W0927 08:59:59.849136    2397 reflector.go:347] object-"openshift-monitoring"/"prometheus-adapter-7m6srg4dfreoi": watch of *v1.Secret ended with: an error on the server ("unable to decode an event from the watch stream: http2: client connection lost") has prevented the request from succeeding`,
+			want: monitorapi.EventInterval{
+				Condition: monitorapi.Condition{
+					Level:   0,
+					Locator: "ns/openshift-monitoring pod/prometheus-adapter-7m6srg4dfreoi uid/ container/",
+					Message: "reason/HttpClientConnectionLost unable to decode an event from the watch stream: http2: client connection lost",
+				},
+				From: time.UnixMicro(1664269199853216).In(time.UTC),
+				To:   time.UnixMicro(1664269199853216).In(time.UTC),
+			},
+		},
+		{
+			name:    "kubelet",
+			logLine: `Sep 27 08:59:59.853216 ci-op-747jjqn3-b3af3-f45pk-worker-centralus2-bdp5s kubenswrapper[2397]: E0927 08:59:59.849143    2397 kubelet_node_status.go:487] "Error updating node status, will retry" err="error getting node \"ci-op-747jjqn3-b3af3-f45pk-worker-centralus2-bdp5s\": Get \"https://api-int.ci-op-747jjqn3-b3af3.ci2.azure.devcluster.openshift.com:6443/api/v1/nodes/ci-op-747jjqn3-b3af3-f45pk-worker-centralus2-bdp5s?timeout=10s\": http2: client connection lost"`,
+			want: monitorapi.EventInterval{
+				Condition: monitorapi.Condition{
+					Level:   0,
+					Locator: "ci-op-747jjqn3-b3af3-f45pk-worker-centralus2-bdp5s",
+					Message: "reason/HttpClientConnectionLost error getting node \"ci-op-747jjqn3-b3af3-f45pk-worker-centralus2-bdp5s\": Get \"https://api-int.ci-op-747jjqn3-b3af3.ci2.azure.devcluster.openshift.com:6443/api/v1/nodes/ci-op-747jjqn3-b3af3-f45pk-worker-centralus2-bdp5s?timeout=10s\": http2: client connection lost",
+				},
+				From: time.UnixMicro(1664269199853216).In(time.UTC),
+				To:   time.UnixMicro(1664269199853216).In(time.UTC),
+			},
+		},
+		{
+			name:    "simple failure",
+			logLine: `Jul 05 17:47:52.807876 ci-op-lxqqvl5x-d3bee-gl4hp-master-0 hyperkube[1495]: I0606 17:47:52.807876    1599 prober.go:121] "Probe failed" probeType="Readiness" pod="openshift-authentication/oauth-openshift-77f7b95df5-r4xf7" podUID=1af660b3-ac3a-4182-86eb-2f74725d8415 containerName="oauth-openshift" probeResult=failure output="Get \"https://10.129.0.12:6443/healthz\": net/http: request canceled while waiting for connection (Client.Timeout exceeded while awaiting headers)"`,
+			want: monitorapi.EventInterval{
+				Condition: monitorapi.Condition{
+					Level:   0,
+					Locator: "ns/openshift-authentication pod/oauth-openshift-77f7b95df5-r4xf7 uid/1af660b3-ac3a-4182-86eb-2f74725d8415 container/oauth-openshift",
```

Comment:
Could we slip node/ into all of these locators?

Reply:
I can if it is the right thing to do but for the Probe error / failures, that code existed already.  The 'status' and 'reflector' cases above are reusing that code.  For the 'kubelet' case I can definitely prefix the locator with 'node/'

Reply:
The code generating that Locator value is https://github.com/openshift/origin/blob/423301d44e82ee4a199a994f3aac4fad548be1ea/pkg/monitor/monitorapi/identification_pod.go#L63-L65

Reply:
Ok the bare string was definitely the one that it looked most useful for above, so that's is fine with me, thinking about it today we could easily work back from a pod name to a node. Thanks!